Library/Styles/SettingsStyles.swift

```diff
@@ -86,7 +86,15 @@ public func settingsContentViewStyle(_ view: UIView) -> UIView {
     |> \.preservesSuperviewLayoutMargins .~ false
 }
 
-public let settingsFooterContentViewStyle = settingsContentViewStyle
+public func settingsFooterContentViewStyle(_ view: UIView) -> UIView {
+  return view
+  |> settingsContentViewStyle
```

Comment:
Can we adjust the spacing here? CMD+X > CMD+V

Reply:
Or Ctrl+I 😄

Reply:
Wuuuuuuuuuuuuuuuut :trollface: